apps/omg_db/config/config.exs

```diff
@@ -3,7 +3,7 @@ use Mix.Config
 # see [here](README.md) for documentation
 
 config :omg_db,
-  type: :leveldb,
+  type: {:system, "DB_TYPE", :leveldb, {String, :to_atom}},
```

Comment:
I just spotted that this config var is undocumented, see https://github.com/omisego/elixir-omg#omgdb-configuration---omg_db-app

Reply:
The question is whether we want to put it there? It's not something we will support indefinitely.

Reply:
Type is only there to ease the transition, no?

Reply:
ah right, nvm then, haven't thought of that